### PR TITLE
chore(agw): Use system libraries for libfluid and lfds710

### LINKED ISSUE #14886: Change dynamically linked libraries to system libraries
# Change dynamically linked libraries to system libraries
- This task is in the context of https://github.com/magma/magma/issues/14885
- CPP libraries that are present in several versions or inconsistent between the Bazel and Make builds should be turned back into system libraries. This applies e.g. to `libfluid_base` and `libfluid_msg`.
- For now only do this if straightforward and document the remainder for https://github.com/magma/magma/issues/14885

### PR DESCRIPTION
**Pull request by LKreutzer**
Signed-off-by: Lars Kreutzer <lars.kreutzer@tngtech.com>

## Summary

- Use system libraries for libfluid and lfds710 instead of git repositories.
  - This is closer to the Make build.
  - See https://github.com/magma/magma/issues/14885 for additional context. 
- Format filenames and linker flags for glog to be consistent. 
- Resolves #14886
- :warning:  Requires https://github.com/magma/magma/pull/14897 for the Bazel workflow to use the updated bazel-base Docker container.

## Test Plan

- [x] Local LTE integ tests
  - [x] Precommit
    - Screenshot from 2023-01-27 11-31-54

  - [x] Extended
    - Screenshot from 2023-01-27 13-06-58
 
- [x] [Sudo python tests](https://github.com/magma/magma/actions/workflows/sudo-python-tests.yml?query=branch%3Asys_lib_bazel)

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by LKreutzer** [comment]
The Bazel workflow is expected to fail until https://github.com/magma/magma/pull/14897 is merged.

**Comment by LKreutzer** [comment]
:warning: Requires https://github.com/magma/magma/pull/14897 to be merged first :warning:

**Comment by LKreutzer** [comment]
Re-based

**Comment by LKreutzer** [comment]
Removed copy step for libfluid patches in Docker build that is not needed anymore.

File: bazel/external/system_libraries.BUILD [hunk 1]
```diff
@@ -76,9 +94,10 @@ cc_library(
 cc_library(
     name = "libglog",
     srcs = glob(
-        ["usr/lib/*-linux-gnu/libglog.so.0"],
+        ["usr/lib/*-linux-gnu/libglog.so"],
```
Comment (nstng) [inline bazel/external/system_libraries.BUILD:97]: Why was this change necessary? Seems unrelated to the fluid and lfds changes.

Comment (LKreutzer) [reply bazel/external/system_libraries.BUILD:97]: > Format filenames and linker flags for glog to be consistent.

This change is not really necessary, but this way the file and linker option specifications are consistent across the libraries in this file.

Comment (nstng) [reply bazel/external/system_libraries.BUILD:97]: the glog pattern is used to expand the asterisk. the asterisk is used because the folder name is different on amd and arm systems. I do not know if it's also called `libglog.so` on arm systems - I would expect that it is consistent, but I do not know without (effort) testing.

Comment (nstng) [reply bazel/external/system_libraries.BUILD:97]: discussed offline - checked that libglog.so is also in the arm package (/)

**Review by nstng — APPROVED** [review]
